models/yiiModels/YiiProjectModel.php

```diff
@@ -180,7 +179,9 @@ public function attributesToArray() {
         $elementForWebService[YiiProjectModel::DESCRIPTION] = $this->description;
         $elementForWebService[YiiProjectModel::OBJECTIVE] = $this->objective;
         $elementForWebService[YiiProjectModel::PARENT_PROJECT] = $this->parentProject;
-        $elementForWebService[YiiProjectModel::WEBSITE] = $this->website;
+        // If website is empty or blank we must be send a null element
```

Comment:
I think this comment would be better in the developer documentation. It's a general remark.